lib/stack_master/parameter_resolvers/s3_object_finder.rb

```diff
@@ -0,0 +1,31 @@
+module StackMaster
+  module ParameterResolvers
+    class S3ObjectFinder
```

Comment:
Can you help me understand why there are two classes? As far as I can tell this class is only ever used by the latest s3 object resolver, so I think it would be more idiomatic to keep this code together.

Reply:
This would allow for multiple types of searching for s3 objects; similar to latest_ami_by_tags and latest_ami.  I plan to add additional resolvers once this is fully working.